include/pisa/score_opt_partition.hpp

```diff
@@ -37,9 +36,8 @@ namespace pisa {
             float sum;
             uint64_t end_sequence;
             uint64_t element_count;
-            float estimated_idf;
 
-            score_window(ForwardIterator begin, posting_t base, wand_cost_t cost_upper_bound, float fixed_cost, size_t size, float e_idf)
+            score_window(ForwardIterator begin, posting_t base, wand_cost_t cost_upper_bound, float fixed_cost, size_t size)
```

Comment:
Honestly, no idea what's happening here, and what you're doing, I'll just assume that removing this `estimated_idf` is intentional and makes sense.

Reply:
Yes, it is done on purpose.